riscv.c

```diff
@@ -790,9 +1264,11 @@ void rv_step(struct riscv_t *rv, int32_t cycles)
 #ifdef ENABLE_COMPUTED_GOTO
 #define OP(instr) &&op_##instr
 #define TABLE_TYPE const void *
+#define TABLE_TYPEC const void *
```

Comment:
I am afraid that `TABLE_TYPEC` is ambiguous to reflect the use of compressed instructions. Can you think of another term?